### PR TITLE
LPS-128284 Fix react warning on MultiPanelSidebarFormsProxy component

### PR DESCRIPTION
**Pull request by interaminense**
Hey guys, this PR is to fix a react warning because it is missing to pass the `ref` to the component
cc: @alinedoleron, @miltonmc

### CODE REVIEW

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebarFormsProxy.es.js [hunk 1]
```diff
@@ -22,32 +22,39 @@ import DragLayer from '../../drag-and-drop/DragLayer.es';
...
 			currentPanelId: 'fields',
 			open: true,
 		});
 
 		return (
-			<DndProvider backend={HTML5Backend} context={window}>
+			<DndProvider
+				backend={HTML5Backend}
+				context={window}
+				ref={forwardRef}
```
Comment (diegonvs) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebarFormsProxy.es.js:56]: We shouldn't never pass a `ref` to a provider component. Why not directly on MultiPanelSidebar?

Comment (interaminense) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebarFormsProxy.es.js:56]: Sure thing! I'll do that.

**Comment by marcelabc** [comment]
I added the label "changes required" because of Diego's comment:

> We shouldn't never pass a ref to a provider component. Why not directly on MultiPanelSidebar?

**Comment by interaminense** [comment]
Done @diegonvs

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**